### DISCUSSION TITLE
Translations are available!!!

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#1077] [Show and tell]
A long requested feature is here: Translations!!!

They are part of the latest release, check it out on https://demo.mediacms.io/ 

The docs if you want to see how to improve it: 
https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#20-translations

**Comment by Mozitux** [comment]
Hi 👋

I want to extend my heartfelt thanks to you and the other contributors. 

Although I have translated my instances multiple times, this feature will now allow me to submit my FR and JP translation (my two native languages) as a pull request. I sincerely hope this functionality will rekindle enthusiasm for MediaCMS and help shine a brighter light on this project.

Warm regards 😃
